src/lib/utils/datetimes.js

```diff
@@ -56,3 +58,36 @@ export const getLocale = (): string => {
   if (navigator.languages !== undefined) return navigator.languages[0];
   return navigator.language ? navigator.language : 'en-US';
 };
+
+/**
+ * gets and transform the browser locale to match the date-fns locale name
+ * e.g.: browser format: `en-US`
+ *       date-fns format: `enUS`
+ *
+ * @returns string the matching locale name, if found, enUS otherwise
+ */
+export const getFormattedLocale = (): string => {
+  const locale = getLocale().split('-');
+  if (locale[1]) {
```

Comment:
This may be clearer if it's a length check instead

Reply:
Updated to check if `locale.length > 1`